release_notes/45-release-notes.adoc

```diff
@@ -0,0 +1,416 @@
+
+* Red Hat has reduced the default timeout setting for {product-title-short} admission controller webhooks from 20 seconds to 10 seconds, resulting in an effective timeout of 12 seconds within the ValidatingWebhookConfiguration. This change aligns with {ocp}'s unconditional cap of 13 seconds. If you rely on longer timeouts, such as those with inline image scanning, you must specify a longer timeout explicitly in the `SecuredCluster` custom resource `admissionControl.timeoutSeconds`, in Helm `admissionControl.dynamic.timeout`, or within a sensor deployment bundle `ValidatingWebhookConfiguration` manifest within the `admission-controller.yaml` file.
+
+* With {product-title-short} 4.5, the ability to snooze Node and Platform CVEs is disabled by default. You must set `ROX_VULN_MGMT_LEGACY_SNOOZE` to `true` on Central to return to the earlier behavior.
+
+[id="documentation-updates-450_{context}"]
+== Documentation updates
+
+* Improved the documentation to clarify that when you update Scanner definitions in offline mode, Scanner retrieves data from Central every 5 minutes, with Central updating the online data every 5 - 20 minutes and the offline data every 3 hours. For more information, see xref:../configuration/enable-offline-mode.adoc#update-scanner-definitions-in-offline-mode[Updating Scanner definitions in offline mode].
+* Updated the documentation to view and configure default policies, manage policy categories, and create your own categories with improved organization and clarity. For more information, see xref:../operating/manage-security-policies.adoc#manage-security-policies[Managing security policies].
+* Updated the documentation to include the `--dnsport` option and updated the example links in the xref:../cli/generating-build-time-network-policies.adoc#using-the-build-time-network-policy-generator_generating-build-time-network-policies[Generating build-time network policies] section.
+* Updated the list of supported operating systems and versions in the xref:../operating/examine-images-for-vulnerabilities.adoc#supported-operating-systems_examine-images-for-vulnerabilities[Supported operating systems] section.
+* Updated the documentation to include new information about using the built-in email notifier in {product-title-managed-short}. For more information, see xref:../integration/integrate-with-email.adoc#integrating-with-email-on-rhacs-cloud-service[Integrating with email on {product-title-managed-short}].
+* Updated managing compliance guide where you can find detailed instructions on how to use the compliance feature in {product-title-short}. For more information, see xref:../operating/manage-compliance/managing-compliance-20.adoc#managing-compliance-20[Managing the compliance 2.0 feature].
```

Comment:
The xref needs to be updated as the compliance feature documentation is currently being restructured.
https://78035--ocpdocs-pr.netlify.app/openshift-acs/latest/operating/manage-compliance/managing-compliance

Reply:
Thanks @agantony The preview build was failing, so I made this change. Once, your PR is merged, I'll updated this link.